cost-model/src/cost_tracker.rs

```diff
@@ -216,7 +216,7 @@ impl CostTracker {
         self.transaction_count.0
     }
 
-    pub fn report_stats(&self, bank_slot: solana_clock::Slot) {
+    pub fn report_stats(&self, bank_slot: solana_clock::Slot, is_leader: bool) {
         // skip reporting if block is empty
```

Comment:
do we still want to skip reporting if the block is empty? I think the original reason for this was to skip all the replay blocks where we didn't use cost-tracker?